code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -419,42 +419,96 @@
 
 /mob/living/carbon/human/proc/shadekin_get_energy()
 	var/datum/species/shadekin/SK = species
+	var/datum/species/crew_shadekin/BESK = species
 
 	if(!istype(SK))
-		return FALSE
+		if(!istype(BESK))
+			return FALSE
 
 	return SK.get_energy(src)
 
 /mob/living/carbon/human/proc/shadekin_get_max_energy()
 	var/datum/species/shadekin/SK = species
+	var/datum/species/crew_shadekin/BESK = species
 
 	if(!istype(SK))
-		return FALSE
+		if(!istype(BESK))
+			return FALSE
 
 	return SK.get_max_energy(src)
 
 /mob/living/carbon/human/proc/shadekin_set_energy(new_energy)
 	var/datum/species/shadekin/SK = species
+	var/datum/species/crew_shadekin/BESK = species
 
 	if(!istype(SK))
-		return FALSE
+		if(!istype(BESK))
+			return FALSE
 
 	SK.set_energy(src, new_energy)
 
 /mob/living/carbon/human/proc/shadekin_set_max_energy(new_max_energy)
 	var/datum/species/shadekin/SK = species
+	var/datum/species/crew_shadekin/BESK = species
 
 	if(!istype(SK))
-		return FALSE
+		if(!istype(BESK))
+			return FALSE
 
 	SK.set_max_energy(src, new_max_energy)
 
 /mob/living/carbon/human/proc/shadekin_adjust_energy(amount)
 	var/datum/species/shadekin/SK = species
+	var/datum/species/crew_shadekin/BESK = species
 
 	if(!istype(SK))
-		return FALSE
+		if(!istype(BESK))
+			return FALSE
 
 	if(amount > 0 || !(SK.check_infinite_energy(src)))
 		var/new_amount = SK.get_energy(src) + amount
 		SK.set_energy(src, new_amount)
+
+/*
```

Comment:
you are handleing it in the lines above, 
get rid of the unessessary commented out code please
(its acceptable to leave commented code out, if it fullfills a different function that the new code you are replacing it with

Reply:
Thanks for pointing it out, removed.